packages/docs/site/docs/01-start-here/03-repos-overview.md

```diff
@@ -0,0 +1,79 @@
+# Understanding the Playground Repositories
+There are a couple of different repos that serve different purposes in the WordPress Playground project. Here's a quick overview of each:
+
+## [WordPress Playground](https://github.com/WordPress/wordpress-playground/)
+This is the main repository for the WordPress Playground project. It contains the core codebase for the project, project boards and issues, and discussions around ongoing project improvements and maintenance.
+
+The repo is there to guide the core contributors to the Playground identify issues, feature requests, and other improvements that need to be made to the project.
+
+Opening issues in the WordPress Playground repository for the "online" version of the Playground, IE: playground.wordpress.net, is a great way to get visibility on a problem or feature request. But there may be a better suited repo below to open the issue, and we'll discuss that shortly.
+
+Searching for issues is a great way to find a fix or workaround, or to see if a feature request has already been made. If you find an issue that matches your problem, you can add a comment to the issue to provide more information or to show that you are also experiencing the same problem.
```

Comment:
Over time many issues will be discussed in the primary repository. If you are new to using Playground and trying it online, try searching for any issue you may be having in the main repo.